### PR TITLE
Make text input optional to SectionBlock

### PR DESCRIPTION
**Pull request by palewire**

### CODE REVIEW

**Comment by palewire** [comment]
For #22

File: slackblocks/blocks.py [hunk 1]
```diff
@@ -58,22 +58,26 @@ class SectionBlock(Block):
...
-        if type(text) is Text:
-            self.text = text
+        if text:
+            if type(text) is Text:
+                self.text = text
+            else:
+                self.text = Text(text)
         else:
-            self.text = Text(text)
+            self.text = text
```
Comment (nicklambourne) [inline slackblocks/blocks.py:73]:
```suggestion
        if not text or type(text) is Text:
            self.text = text
        else:
            self.text = Text(text)
```

**Review by nicklambourne — CHANGES_REQUESTED** [review]
Apologies for the delay in getting to this, what do you think about my suggestion for simplifying it a little?

File: slackblocks/blocks.py (hunk 1 above)
Comment (palewire) [reply slackblocks/blocks.py:73]: Looks good to me.

**Comment by palewire** [comment]
Love it.

**Pull request merged**